script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_2.txt

```diff
@@ -85,7 +85,7 @@ I can't remember exactly when it happened, but if Yumizuka and I had ever talked
 "Well, I'd better get going. Is Habano-sensei in the staff room?"
 "Yes, he said he would deduct points if you didn't show in ten minutes. I have always wondered why they deduct points for things like that."
 <Page25>
-Yumizuka laughs awkwardly, without any sarcasm or irony.
+Yumizuka laughs awkwardly, without any sarcasm nor irony.
```

Comment:
Not sure this is correct - I think that "with neither sarcasm nor irony" would be valid, but this is the positive form so I think `or` is right? @yk1512 is grammar expert

Reply:
> 嫌味でも皮肉でもなく、おかしそうに弓塚は笑う

'with neither sarcasm nor irony' fits more since its negative in jp